homeassistant/components/aurora/config_flow.py

```diff
@@ -0,0 +1,130 @@
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+
+        if user_input is not None:
+            name = user_input[CONF_NAME]
+            longitude = user_input[CONF_LONGITUDE]
+            latitude = user_input[CONF_LATITUDE]
+
+            session = aiohttp_client.async_get_clientsession(self.hass)
+            api = AuroraForecast(session=session)
+
+            try:
+                await api.get_forecast_data(longitude, latitude)
+            except ConnectionError:
+                errors["base"] = "cannot_connect"
+            except Exception:  # pylint: disable=broad-except
+                _LOGGER.exception("Unexpected exception")
+                errors["base"] = "unknown"
+            else:
+                await self.async_set_unique_id(
+                    f"{DOMAIN}_{user_input[CONF_LONGITUDE]}_{user_input[CONF_LATITUDE]}"
+                )
+                self._abort_if_unique_id_configured()
+                return self.async_create_entry(
+                    title=f"Aurora - {name}", data=user_input
+                )
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_NAME, default=DEFAULT_NAME): str,
+                    vol.Required(
+                        CONF_LONGITUDE,
+                        default=self.hass.config.longitude,
+                    ): vol.All(
+                        vol.Coerce(float),
+                        vol.Range(min=-180, max=180),
+                    ),
+                    vol.Required(
+                        CONF_LATITUDE,
+                        default=self.hass.config.latitude,
+                    ): vol.All(
+                        vol.Coerce(float),
+                        vol.Range(min=-90, max=90),
+                    ),
+                }
+            ),
+            errors=errors,
+        )
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle options flow changes."""
+
+    def __init__(self, config_entry):
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None):
+        """Manage options."""
+
+        if user_input is not None:
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(
+                        CONF_THRESHOLD,
+                        default=self.config_entry.options.get(
+                            CONF_THRESHOLD, DEFAULT_THRESHOLD
+                        ),
+                    ): vol.All(
+                        vol.Coerce(int),
+                        vol.Range(min=0, max=100),
+                    ),
+                    vol.Required(
+                        CONF_SCAN_INTERVAL,
```

Comment:
Let's not make this configurable

Reply:
OK - will remove scan interval configurability.

Reply:
Will remove the configurable scan interval.